src/engine/playback.cpp

```diff
@@ -1590,9 +1631,17 @@ bool DivEngine::nextTick(bool noAccum, bool inhibitLowLat) {
           if (chan[i].volSpeed!=0) {
             chan[i].volume=(chan[i].volume&0xff)|(dispatchCmd(DivCommand(DIV_CMD_GET_VOLUME,i))<<8);
             chan[i].volume+=chan[i].volSpeed;
-            if (chan[i].volume>chan[i].volMax) {
+            if (chan[i].volSpeedTarget!=-1 && (chan[i].volume==chan[i].volSpeedTarget || (chan[i].volume>chan[i].volSpeedTarget)==(chan[i].volSpeed>0))) {
```

Comment:
I have a feeling this will glitch up at some point. Is it possible to make this code more readable? (`==` will make volume miss volSpeedTarget 90% of the time, and the check afterwards is hard to comprehend